templates/opensshd.conf.j2

```diff
@@ -190,12 +189,25 @@ GatewayPorts no
 X11Forwarding no
 X11UseLocalhost yes
 
-# Look up the remote host name, defaults to false from 6.8, see: http://www.openssh.com/txt/release-6.8
-UseDNS {{ 'yes' if ssh_use_dns else 'no' }}
+# User environment configuration
+# ==============================
+
+{% if ssh_server_permit_environment_vars %}
+PermitUserEnvironment yes
+{% for item in ssh_server_permit_environment_vars %}
+AcceptEnv {{ item }}
+{% endfor %}
+{% else %}
+PermitUserEnvironment no
+{% endif %}
 
 # Misc. configuration
 # ===================
 
+Compression {{ 'yes' if ssh_compression else 'no' }}
```

Comment:
`ssh_compression` is (supposed to be) a boolean so the [`bool` Jinja2 filter in Ansible](https://docs.ansible.com/ansible/playbooks_filters.html#other-useful-filters) should be used to ensure correct evaluation even if users do not behave :wink: Same with other boolean vars.